src/rtctools_heat_network/heat_mixin.py

```diff
@@ -200,20 +221,20 @@ def _get_min_bound(bound):
 
         # Mixed-integer formulation applies only to hot pipes, not to cold
         # pipes.
-        for p in self.hot_pipes:
-            flow_dir_var = f"{p}__flow_direct_var"
+        for pc in self.hot_pipes:
```

Comment:
why do we change from p to pc here in the naming?

Reply:
That was just a mistake, good find :)